api/auth/auth-router.js

```diff
@@ -14,11 +16,24 @@ router.post("/register", validateRoleName, (req, res, next) => {
       "role_name": "angel"
     }
    */
-});
 
+router.post("/register", validateRoleName, (req, res, next) => {
+  const { username, password, role_name } = req.body;
+
+  Users.add({
+    username,
+    password: bcrypt.hash(password, 20),
+    role_name,
```

Comment:
daca observi in validateRoleName tu scrii role_name direct pe request. De ce se face asta? Nu ar trebui sa utilizezi acea valoare in loc de ceea ce primesti in body?

Reply:
Da, aveti dreptate, am modificat, dar asta nu schimba nimic:
Users.add({
    username,
    password: bcrypt.hash(password, 20),
    role_name: req.roleName,
  })